src/utils/getInstru.js

```diff
@@ -0,0 +1,5 @@
+export function getInstru(videos) {
```

Comment:
Instead of writing 3 different functions, you can create a generic function as the code is repeating and then check the condition with the payload value.

Reply:
Good suggestion.Thank you